webdriver/tests/support/asserts.py

```diff
@@ -81,8 +83,12 @@ def assert_response_headers(headers):
     """
     assert 'cache-control' in headers
     assert 'no-cache' == headers['cache-control']
-    assert 'content-type' in headers
-    assert 'application/json; charset=utf-8' == headers['content-type']
+    if PY3:
```

Comment:
I think since we've moving from less strict to stricter behavior, this seems reasonable (e.g. as opposed to just forcing the dict to lower-case). Can you add a comment that explains why this is necessary, e.g.

```
    # In Python 2, HTTPResponse normalizes header keys to lowercase, whereas
    # Python 3 preserves the case. See
    # https://github.com/web-platform-tests/wpt/pull/22858#issuecomment-612656097
```